saltstack/pillar/custom/devel_standalone.sls

```diff
@@ -0,0 +1,13 @@
+{# Custom Settings #}
+
+{% set branch = 'devel' %} {# Use 'release' or 'devel' #}
+{% set version = '3.19' %}
+{% set environment = 'dev' %} {# Use 'dev' or 'prod' #}
+{% set r_download = 'https://cran.r-project.org/src/base-prerelease/R-devel.tar.gz' %}
```

Comment:
It might be better to use the daily devel tarball at https://stat.ethz.ch/R/daily/R-devel.tar.gz, which is what I typically install on the machines. Previously, prerelease was only updated just before the release but they seem to be updating it regularly; however, we've also recommended this for the external builders and it might be better to keep everything using the same tarball.

Reply:
Thank you! Changed!